spec/enumerable_spec.rb

```diff
@@ -0,0 +1,111 @@
+require '.\lib\enumerable.rb'
```

Comment:
probably better to use `require './lib/enumerable.rb` as '.\lib\enumerable.rb' throws error in MacOS